zcbor build error when ZCBOR_VERBOSE is set

**Describe the bug**
Upon trying to build smp_svr, where ZCBOR_VERBOSE is set by default, the build fails with errors like:
```
In file included from /tmp/aa/modules/lib/zcbor/include/zcbor_decode.h:12,
                 from /tmp/aa/modules/lib/zcbor/src/zcbor_decode.c:11:
/tmp/aa/modules/lib/zcbor/src/zcbor_decode.c:889:24: error: expected ')' before 'PRIuFAST32'
  889 |  zcbor_print("Found %" PRIuFAST32 " elements.\r\n", max_decode);
      |                        ^~~~~~~~~~
/tmp/aa/modules/lib/zcbor/include/zcbor_common.h:50:33: note: in definition of macro 'zcbor_print'
   50 | #define zcbor_print(...) printk(__VA_ARGS__)
      |                                 ^~~~~~~~~~~
/tmp/aa/modules/lib/zcbor/src/zcbor_decode.c:889:14: warning: spurious trailing '%' in format [-Wformat=]
  889 |  zcbor_print("Found %" PRIuFAST32 " elements.\r\n", max_decode);
      |              ^~~~~~~~~
/tmp/aa/modules/lib/zcbor/include/zcbor_common.h:50:33: note: in definition of macro 'zcbor_print'
   50 | #define zcbor_print(...) printk(__VA_ARGS__)
      |                                 ^~~~~~~~~~~
/tmp/aa/modules/lib/zcbor/src/zcbor_decode.c:889:21: note: format string is defined here
  889 |  zcbor_print("Found %" PRIuFAST32 " elements.\r\n", max_decode);
      |                     ^
[191/213] Building C object zephyr/kernel/CMakeFiles/kernel.dir/system_work_q.c.obj
ninja: build stopped: subcommand failed.
```

**Expected behavior**
No build errors

**Impact**
Annoyance

**Environment (please complete the following information):**
 - OS: Linux
 - Toolchain: Zephyr SDK 0.14.0
 - Commit SHA or Version used: fc88ddf59cc2814ee88ed39a60e4673d3a13a2a6

--- a/modules/zcbor/Kconfig
+++ b/modules/zcbor/Kconfig
@@ -26,6 +26,9 @@ config ZCBOR_STOP_ON_ERROR
 config ZCBOR_VERBOSE
 	bool "Make zcbor code print messages"
 
+	# TODO: FIXME: Remove when Minimal LIBC has been fixed.
+	depends on !MINIMAL_LIBC # Because Minimal LIBC doesn't have PRIuFAST32 and PRIxFAST32.
+
 config ZCBOR_ASSERT
 	def_bool ASSERT
 

